pkgs/development/python-modules/trustme/default.nix

```
@@ -23,7 +23,7 @@ buildPythonPackage rec {
   checkInputs = [
     service-identity
     pytestCheckHook
-  ] ++ lib.optionals (!stdenv.isDarwin && !stdenv.isAarch64) [
+  ] ++ lib.optionals (!stdenv.isDarwin) [
```

This is meant to be not aarch64-darwin. Switch the && to an ||. My apologies; I didn't clock the change in the review of my PR.

(Because as we all know, !(A && B) is !A || !B, and yet we still make this slip)
```suggestion
  ] ++ lib.optionals (!stdenv.isDarwin || !stdenv.isAarch64) [
```